sopel/modules/tell.py

```diff
@@ -65,7 +67,26 @@ def dumpReminders(fn, data, lock):
     return True
 
 
+class FormatSection(StaticSection):
+    tell_format = ValidatedAttribute(
+        'tell_format',
+        default='relative')
+    """Show relative timestamps by default"""
```

Comment:
```suggestion
    """Which type of timestamp to use when delivering messages."""
```

I don't necessarily mean for this to be the exact wording, but something closer to this or to the prompt used in `configure_setting()` below would be much better.